lib/recase/cases/generic.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Recase.Generic do
+  @moduledoc """
+  Generic module to split and join strings back.
+
+  This module should not be used directly.
+  """
+
+  @splitters Application.get_env(:recase, :delimiters, :symbol)
+
+  @delimiters (case @splitters do
+                 list when is_list(list) ->
+                   list
+
+                 :symbol ->
+                   [all, down, up] = Enum.map([32..127, ?a..?z, ?A..?Z], &Enum.to_list/1)
```

Comment:
This treats digits as delimiters as well; maybe it should eliminate digits as well as letters from the list of delimiters.